third_party/nvfuser/csrc/dynamic_type.h

```diff
@@ -22,37 +23,24 @@ class TORCH_CUDA_CU_API EvaluatorValue {
   explicit EvaluatorValue(size_t i) : value_((int64_t)i) {}
   EvaluatorValue() : EvaluatorValue(0) {}
 
-  // Avoid using copy constructor of c10::variant as it's
-  // deprecated.
-  EvaluatorValue(const EvaluatorValue& other) {
-    value_ = other.value_;
-  }
-
-  // Explicitly define copy assignment operator as its implicit definition is
-  // deprecated
-  EvaluatorValue& operator=(const EvaluatorValue& other) {
-    value_ = other.value_;
-    return *this;
-  }
-
```

Comment:
The functions were there to workaround `c10::variant` bug, and it is not needed anymore if we use `std::variant`